msal/application.py

```diff
@@ -139,17 +142,23 @@ def __init__(
         :param TokenCache cache:
             Sets the token cache used by this ClientApplication instance.
             By default, an in-memory cache will be created and used.
+        :param http_client: (optional)
+            Your implementation of abstract class HttpClient <msal.oauth2cli.http.http_client>
+            Defaults to a requests session instance
         :param verify: (optional)
             It will be passed to the
             `verify parameter in the underlying requests library
+            This does not apply if you have chosen to pass your own Http client
             <http://docs.python-requests.org/en/v2.9.1/user/advanced/#ssl-cert-verification>`_
```

Comment:
```suggestion
            <http://docs.python-requests.org/en/v2.9.1/user/advanced/#ssl-cert-verification>`_
            This does not apply if you have chosen to pass your own Http client.
```
Nit. Let's swap these two lines. Because, albeit unfortunately unobvious, the original `verify parameter ... <that long url>` part was actually a text with link. Our new sentence should better be NOT part of the link text. Similar suggestion to the `proxies` and `timeout`.